### ISSUE TITLE
Clicking on CTA multiple times returns multiple error messages

### ISSUE THREAD

**Issue by machikoyasuda** [reactioncommerce/reaction#3670]
### Expected behavior
Only one error box will show up, no matter how many times you erroneously click the CTA without choosing a variant.

### Actual behavior
Multiple error boxes show up

<img width="689" alt="screen shot 2018-02-08 at 11 30 42 am" src="https://user-images.githubusercontent.com/3673236/35993915-b0ab2326-0cc3-11e8-85ef-aada418eb811.png">

### Steps to reproduce the behavior
1. Click on CTA without clicking a variant
2. Click it again and again and again...

### Versions

This is not a _new_ bug in 1.8. This is a bug existing in 1.7.  Test here: https://swag.getreaction.io/product/reaction-bag-of-beans

```
Node: 8.2.1
NPM: 5.6.0
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.27.0
Reaction: 1.8.0
Reaction branch: release-1.8.0
Docker: 17.09.0-ce
```

Browser: http://whatismybrowser.com/w/KE449ZE

### QA

QA by @willopez + @machikoyasuda

**Comment by spencern** [comment]
@rymorgan is this the correct functionality?

**Comment by rymorgan** [comment]
I think it should only come up once. But, I think they better UX solution is to show the CTA with a disabled state until they make the required selection.

**Comment by zenweasel** [comment]
I don't know that we should ever put bugs in the Icebox?

**Comment by spencern** [comment]
@zenweasel that's fair. I put it there while waiting for a response from @rymorgan, but moving it the priority list may have been a better choice.

**Comment by zenweasel** [comment]
I think the process now is to put stuff we need design input in the "Design" column? I could be wrong.

**Comment by spencern** [comment]
I don't remember. Where does something go from that column once the design is complete?

**Comment by zenweasel** [comment]
Back to where it came from I suppose

**Comment by rymorgan** [comment]
Yea, I think that's right. If it needs design, the best way for us would be to put it in the design column and tag us. 

I had responded earlier in this thread but actually looking at this again, I think there is a better solution to this alert @cassytaylor is going to quickly mock it up.

**Comment by cassytaylor** [comment]
Hey all, our proposal is to show the loading spinner in the CTA button when clicked, then animate in the error alert (brief fade in + slide down) to its current position. If the user clicks the CTA button again, the loading spinner would appear (to give feedback), and the error alert would repeat its animation (basically, re-load in the same spot.) This would happen as many times as the user triggers the error. 
Video mock - https://drive.google.com/file/d/1azaCGbmLftwt6Zhj-IiLpc1voUABL2qI/view?usp=sharing

**Comment by aldeed** [comment]
Fixed in 2.0.0
